releasenotes/notes/add-graphql-core-support-fd20dcbebfcaa8c7.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    graphql: add tracing for ``graphql-core>2``. See `the documentation <https://ddtrace.readthedocs.io/en/stable/integrations.html#graphql>`_
```

Comment:
I think this is causing issues with sphinx/rST.

Reply:
Maybe it should link to ‘graphql-core’ instead